app/drivers/uart.c

```diff
@@ -116,8 +122,23 @@ void HAL_UART_RxCpltCallback(UART_HandleTypeDef *huart)
 {
 	static BaseType_t xHigherPriorityTaskWoken = pdFALSE;
 
-	xSemaphoreGiveFromISR(self.rx_done_semaphore, &xHigherPriorityTaskWoken);
+	xQueueSendFromISR(self.uart_rx_queue_handle, &self.uart_rx_item, &xHigherPriorityTaskWoken);
 	portYIELD_FROM_ISR(xHigherPriorityTaskWoken);
+	HAL_UART_Receive_DMA(&self.uart_handle, (uint8_t*) self.uart_rx_item.data, USB_FS_MAX_PACKET_SIZE);
+}
+
+void HAL_UART_AbortReceiveCpltCallback (UART_HandleTypeDef *huart)
```

Comment:
Remove the space before "(".

Also we should rename huart to p_uart in this file.